src/components/AddBook.jsx

```diff
@@ -32,19 +32,18 @@ const AddBook = () => {
   ];
 
   return (
-    <section>
+    <section className="add-book-section">
+      <div className="form-title">ADD NEW BOOK</div>
       <form onSubmit={addBookToList}>
-        <div>
-          <input type="text" name="title" id="name" value={title} placeholder="Title" onChange={(e) => setTitle(e.target.value)} required />
-          <input type="text" name="author" id="author" value={author} placeholder="Author" onChange={(e) => setAuthor(e.target.value)} required />
-          <select value={category} onChange={(e) => setCategory(e.target.value)} required>
-            <option value="" disabled>Select a category</option>
-            {Categories.map((categorie) => (
-              <option key={categorie} value={categorie}>{categorie}</option>
-            ))}
-          </select>
-          <button type="submit"> Add Book </button>
-        </div>
+        <input type="text" name="title" id="name" value={title} placeholder="Book title" onChange={(e) => setTitle(e.target.value)} required />
+        <input type="text" name="author" id="author" value={author} placeholder="Book author" onChange={(e) => setAuthor(e.target.value)} required />
+        <select value={category} onChange={(e) => setCategory(e.target.value)} required>
+          <option value="" disabled>Select a category</option>
+          {Categories.map((categorie) => (
+            <option key={categorie} value={categorie}>{categorie}</option>
+          ))}
+        </select>
+        <button className="add-btn" type="submit"> Add Book </button>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirement, As per the project requirement your design should match exactly with the **Zeplin's** design. Your button is not aligned properly with the rest of the content of the website, Please carefully look at the alignment of your `header` logo and `Add Book` button. Both are not aligned properly. Kindly reduce the spacing or adjust your style to match the design.

### Requirement
![image](https://github.com/mulugithub/bookstore/assets/66476329/e249a748-d533-4044-bb32-f21d4e2a6001)

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://github.com/mulugithub/bookstore/assets/66476329/f376a025-75ec-4acf-914b-aa7801e3c3f5)

</td>
<td>

![Screenshot from 2023-09-09 21-55-19](https://github.com/mulugithub/bookstore/assets/66476329/c2f6911b-ff1a-403b-a744-565e1b9bf90e)

</td>
</tr>
</table>